### PR TITLE
[win10] added support for uwp build

### PR DESCRIPTION
**Pull request by afedchin**

### CODE REVIEW

**Comment by notspiff** [comment]
We should setup appveyor here as well.

**Comment by afedchin** [comment]
yes, we should, but this doesn't touch anything except uwp. we will see details on nightly build

**Comment by notspiff** [comment]
Didnt mean to imply this was held back by it just noticed it. Pr incoming

11. nov. 2017 17:38 skrev "Anton Fedchin" <notifications@github.com>:

**Comment by Rechi** [comment]
Why duplication `depends/windows` to `depends/windowsstore` instead of adding the additional changes in depends/windows?

**Comment by afedchin** [comment]
on windows we build nasm, on windowsstore I download precompiled version. Also I prefer to don't touch other platforms.

File: depends/windowsstore/nasmw/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+project(nasmw)
+
+cmake_minimum_required(VERSION 2.8)
+set(NASMW_ARCHIVE_NAME nasm-2.12.02)
+
+message(STATUS "Downloading ${NASMW_ARCHIVE_NAME}-win32.zip")
+file(DOWNLOAD 
+	http://www.nasm.us/pub/nasm/releasebuilds/2.12.02/win32/${NASMW_ARCHIVE_NAME}-win32.zip
+	${CMAKE_CURRENT_BINARY_DIR}/${NASMW_ARCHIVE_NAME}-win32.zip
+	SHOW_PROGRESS
```
Comment (Rechi) [inline depends/windowsstore/nasmw/CMakeLists.txt]: why not handle download in `nasmw.txt`?

Comment (afedchin) [reply depends/windowsstore/nasmw/CMakeLists.txt]: good idea, thanks

**Pull request merged**